LibraryApp/src/redux/books/reducer.js

```diff
@@ -4,11 +4,17 @@ import { createReducer, completeReducer, completeState } from 'redux-recompose';
 import { actions } from './actions';
 
 const stateDescription = {
-  books: []
+  books: [],
+  searchString: ''
 };
 
-const initialState = completeState(stateDescription);
+const initialState = completeState(stateDescription, ['searchString']);
```

Comment:
why is this `searchString` in here?

Reply:
i needed the input value to make the filter in the search screen with selectors